### DISCUSSION TITLE
Please reconsider the release process

### DISCUSSION THREAD

**Discussion by xzel23** [spotbugs/spotbugs-gradle-plugin#1204] [General]
I barely can keep up with updating the SpotBugs plugin lately. Maybe there has been a change in the release pipeline that automatically does a new release whenever a PR is merged. But IMHO this just creates a bunch of useless releases that bring no benefit to any of the spotbugs users.

Let's look at the 6.0.x releases:
6.0.19: -1, gradle dependency update
6.0.18: +1, Spotbugs update
6.0.17: -1, gradle dependency update
6.0.16: -1, gradle dependency update
6.0.15: -1, gradle dependency update
6.0.14: +1, Spotbugs update
6.0.13: -1, gradle dependency update
6.0.12: -1, archunit dependency update
6.0.11: -1, gradle dependency update
6.0.10: +1, Spotbugs update
6.0.9: -1, gradle dependency update
6.0.8: -1, gradle dependency update
6.0.7: -1, gradle dependency update
6.0.6: +1, bugfix
6.0.5: -1, gradle dependency update
6.0.4: +1, bugfix
6.0.3: +1, Spotbugs update
6.0.2: -1, archunit dependency update
6.0.1: -1, gradle dependency update
6.0.0: +1, major release

That's 20 releases, one being the major release, four spotbugs updates, two bugfix updates, and 13 dependency updates.

And as far as I can see, all these dependency updates only make a difference during build time. It would be another thing if runtime dependencies were updated that fix CVEs or bugs.

That's 13 releases that have zero impact on the plugin functionality! Please reconsider releasing all these pointless minor releases.

**Comment by hazendaz** [comment]
@KengoTODA How exactly is this repo releasing?  I've looked through gradle files best I could (not really a gradle user) and the GHA files and do not see how it happens.  I believe due to 'fix' being put automatically on pull requests coming from renovate but we don't have control over how those are flagged best I can tell and I 100% agree here. Its a complete shock every time this releases.  Its doing so way too frequently.  Only releases here that should have been considered were: We did something that we feel requires a release or spotbugs upgrade.  Outside of that I see no value in releases occurring like this.

**Reply by hazendaz** [reply]
Ok I got it now.  I'll replace fix with chore anytime I see it, and it's nothing important to cause a release.  I'm closing this as understood how it's occurring and how to prevent it.
